src/plots/Mesh/MeshAttributes.code

```diff
@@ -189,3 +187,30 @@ MeshAttributes::ProcessOldVersions(DataNode *parentNode,
             searchNode->RemoveNode("lineStyle");
     }
 }
+
+Target: xml2info
```

Comment:
This method contains no custom logic, so isn't needed.

Reply:
I'm confused. I didn't touch this method and don't understand the rationale for removing it if that is what you are requesting. It seems to me like it is custom to handle reading of mesh attributes that pre-date 2.0.0 and 3.0.0.

Reply:
I apologize if the positioning of my comment caused confusion. It's referring to 'InitializePlotAtts' which you added to the code file.

Reply:
InitializePlotAtts still needs to be removed from the .code file.